src/main/java/core/inventory/items/Dagger.java

```diff
@@ -22,12 +22,14 @@
     private final int range;
     private final int damage;
     public UUID id;
+    static final int DAGGER_DAMAGE = 2;
+    static final int DAGGER_RANGE = 1;
 
     public Dagger(int level) {
         this.level = level;
         this.size = this.level;
-        this.range = this.level;
-        this.damage = this.level * 2;
+        this.range = this.level * DAGGER_RANGE;
+        this.damage = this.level * DAGGER_DAMAGE;
```

Comment:
i think that multiplying by level makes it still a magic number, should at least rename `DAGGER_DAMAGE` to `DAGGER_DAMAGE_PER_LEVEL` then we know what the variable is actually doing, same for the others in `dagger` and `sword`